src/packages/ganache/README.md

```diff
@@ -476,7 +476,43 @@ Server:
   -p, --server.port, --port             Port to listen on.
                                         deprecated aliases: --port
                                                                                       [number] [default: 8545]
+```
+
+### Detached Instances
+
+Ganache can be started as a background instance via the CLI by providing the following argument (along with any combination
+of the Ganache startup arguments above):
+
+```
+  -D, --detach, --😈                     Run Ganache in detached (daemon) mode.                       [boolean]
+```
+
+This will start Ganache as a background process, and return to the console as soon as Ganache has started and ready to
+receive requests. A friendly name will be returned to STDOUT which can then be used to interact with the instance via
+the `ganache instances` command with the following arguments:
+
+```
+Commands:
```

Comment:
I think that's a great idea. I found myself typing `ganache instances...` when trying to start a new instance, so this would definitely be an improvement.

I made https://github.com/trufflesuite/ganache/issues/3903 to cover that.